care/facility/api/viewsets/file_upload.py

```diff
@@ -18,7 +24,7 @@ class FileUploadFilter(filters.FilterSet):
 
 
 class FileUploadViewSet(
-    CreateModelMixin, RetrieveModelMixin, ListModelMixin, GenericViewSet,
+    CreateModelMixin, RetrieveModelMixin, ListModelMixin, UpdateModelMixin, DestroyModelMixin, GenericViewSet,
```

Comment:
The update should use a different serializer and not update anything to AWS.

Reply:
updated it